tests/gcp/hooks/datastore.py

```diff
@@ -50,12 +52,12 @@ def test_allocate_ids(self, mock_get_conn):
         self.datastore_hook.connection = mock_get_conn.return_value
         partial_keys = []
 
-        keys = self.datastore_hook.allocate_ids(partial_keys)
+        keys = self.datastore_hook.allocate_ids(partial_keys=partial_keys, project_id=GCP_PROJECT_ID)
```

Comment:
Can you also add tests when project ID is missing?

Reply:
Sure, should I duplicate all test or create just one case?

Reply:
I think one is enough for each hook method. But we should check the complete lack - both in connection and in parameters.